app/src/components/keyCloakServiceClientMgr.js

```diff
@@ -169,17 +169,23 @@ class KeyCloakServiceClientManager {
     }
   }
 
-  async fetchAllClients() {
-    log.info('KeyCloakServiceClientManager.fetchAllClients');
+  async fetchAllClients(kcEnv) {
+
+    log.info('KeyCloakServiceClientManager.fetchAllClients in ' + kcEnv);
```

Comment:
log.debug('KeyCloakServiceClientManager.fetchAllClients',  `Env: ${kcEnv}`);

Reply:
I no longer have kcEnv parameter passed to this function.